tools/project_manager/show_word_context.php

```diff
@@ -11,6 +11,7 @@
 include_once($relPath.'misc.inc'); // array_get(), get_integer_param()
 include_once('./post_files.inc');
 include_once("./word_freq_table.inc");
+include_once($relPath.'page_controls.inc');
```

Comment:
Move this up next to the other `$relPath` includes.

Reply:
Done this.